nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/GetAzureEventHub.java

```diff
@@ -275,11 +279,14 @@ public void tearDown() throws ProcessException {
             if (null != eventHubClient) {
                 eventHubClient.closeSync();
             }
-        } catch (final ServiceBusException e) {
+            executor.shutdown();
```

Comment:
if event hub client close throws exception this will be skipped.  it should be in its own block